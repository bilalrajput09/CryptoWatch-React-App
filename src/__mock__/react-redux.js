const useSelector = jest.fn();

export default { useSelector };
