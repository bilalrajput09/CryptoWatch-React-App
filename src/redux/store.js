import { configureStore } from '@reduxjs/toolkit';
import coinsSlice from './coinsSlice';

const store = configureStore({
  reducer: { coins: coinsSlice },
});

export default store;
