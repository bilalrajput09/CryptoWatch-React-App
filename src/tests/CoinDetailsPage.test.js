import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';

test('render DetailsPage correctly', () => {
  const componentTree = renderer
    .create(<Provider store={store} />)
    .toJSON();
  expect(componentTree).toMatchSnapshot();
});
