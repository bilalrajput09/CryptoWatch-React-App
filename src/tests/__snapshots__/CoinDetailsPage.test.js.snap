// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render DetailsPage correctly 1`] = `null`;
